### PR TITLE
[zh-cn] sync reference/labels-annotations-taints/_index.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -798,6 +833,31 @@ This annotation will be added to dynamic provisioning required PVC.
...
+
+The value of the annotation isn't important; if this annotation exists on a node,
+then storage attaches and detaches are controller managed.
+-->
+### volumes.kubernetes.io/controller-managed-attach-detach   {#volumes-kubernetes-io-controller-managed-attach-detach}
+
+用于：Node
+
+如果节点已在其自身上设置了注释 `volumes.kubernetes.io/controller-managed-attach-detach`
+，那么它的存储挂接和解除挂接的操作是交由运行在
```
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:855]:
```suggestion
如果节点已在其自身上设置了注释 `volumes.kubernetes.io/controller-managed-attach-detach`，
那么它的存储挂接和解除挂接的操作是交由运行在
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -314,6 +314,41 @@ This label has been deprecated. Please use `kubernetes.io/os` instead.
...
+用于：APIService
+
+`kube-apiserver` 会在由 API 服务器自动创建的任何 APIService 对象上设置这个标签。
+该标签标记了控制平面应如何管理该 APIService。你不应自行添加、修改或删除此标签。
+
+{{< note >}}
+<!--
+Automanaged APIService objects are deleted by kube-apiserver when it has no built-in or custom resource API corresponding to the API group/version of the APIService.
+-->
+当 Automanaged APIService 对象没有与 APIService 的 API 组/版本对应的内置或自定义资源 API 时，
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:339]:
```suggestion
当自动托管的 APIService 对象没有内置或自定义资源 API 对应于该 APIService 的 API 组/版本时，
```
这样会不会通顺点

**Comment by windsonsea** [comment]
Could you include those changes recently merged? You can check details by running:

```
./scripts/lsync.sh content/zh-cn/docs/reference/labels-annotations-taints/_index.md
```

image

**Comment by Sea-n** [comment]
Hey @gaogao101,

Can you sync with the latest version?
Please run the command @windsonsea mentioned, there are PR #37928, #36399, #39122 and #39367 needs to be included.

**Comment by windsonsea** [comment]
More than 2 months passed. This PR will be closed and might be taken over by other contributors. You can re-open it at any time when you are available, thanks.

**Pull request closed without merging**